native-shared-library/README.md

```diff
@@ -60,8 +59,12 @@ This example demonstrates how to use the **Native Image C API**. We will:
 
 4. Compile the _main.c_ using `clang`.  
     ```bash
-    $JAVA_HOME/languages/llvm/native/bin/clang -I ./ -L ./ -l envmap -Wl,-rpath ./ -o main main.c 
+    $JAVA_HOME/bin/clang -I ./ -L ./ -l envmap -Wl,-rpath ./ -o main main.c 
```

Comment:
There is no `$JAVA_HOME/bin/clang`  in the GraalVM installation.

Reply:
See the LLVM documentation https://www.graalvm.org/latest/reference-manual/llvm/#llvm-toolchain. 
Or recommend to install LLVM toolchain and compile _main.c_ with it. Or recommend to use other `clang` distribution.